src/reference/asciidoc/aggregator.adoc

```diff
@@ -117,11 +117,15 @@ NOTE: In the interest of code simplicity and promoting best practices such as lo
 Starting with version 5.1, after processing message group, an `AbstractCorrelatingMessageHandler` performs a `MessageBuilder.popSequenceDetails()` message headers modification for the proper splitter-aggregator scenario with several nested levels.
 It is done only if the message group release result is not a message or collection of messages.
 In that case a target `MessageGroupProcessor` is responsible for the `MessageBuilder.popSequenceDetails()` call while building those messages.
-Strting with version 5.3 if sequenceDetails of the outputMessage matches with first message of group, `MessageBuilder.popSequenceDetails()` will be performed.
+
+Additionally, Starting with version 5.3, if `MessageGroupProcessor` returns a Message, `MessageBuilder.popSequenceDetails()` will be performed on the `outputMessage`  if the sequenceDetails matches with first message of group.
+
 This functionality can be controlled by a new `popSequence` `boolean` property, so the `MessageBuilder.popSequenceDetails()` can be disabled in some scenarios when correlation details have not been populated by the standard splitter.
 This property, essentially, undoes what has been done by the nearest upstream `applySequence = true` in the `AbstractMessageSplitter`.
 See <<./splitter.adoc#splitter,Splitter>> for more information.
 
+
+
```

Comment:
What are these blank lines for?

Reply:
I thought it would be better readable with blank lines in between

Reply:
Yeah... And why haven't you changed the whole doc for the same style? 😉 
Where is consistency? 

I mean it's OK to have only one line between paragraphs: the doc tool is going to format it for us properly.
More over the final decision about formatting is left on us, project owner. So, you can change something for code style and formatting, but we may reject it.